ert3/console/_console.py

```diff
@@ -77,6 +90,37 @@ def _build_argparser():
     return parser
 
 
+def _init(args):
+    assert args.sub_cmd == "init"
+
+    if args.example is None:
+        ert3.workspace.initialize(Path.cwd())
+    else:
+        _EXAMPLE_NAME = args.example
```

Comment:
This could be named simply example_name. Underscore indicates it is private, but all locals are private. Capital letters indicate it is a shared constant, but it is a local to only this function.

Reply:
my bad